aten/src/ATen/native/TensorShape.cpp

```diff
@@ -3846,56 +3877,64 @@ at::Tensor lift_fresh(const at::Tensor& self) {
 
 at::Tensor& _fw_primal_copy_out(const at::Tensor & self, int64_t level, at::Tensor & out) {
   auto tmp = self._fw_primal(level);
-  out.copy_(tmp);
+  resize_out_helper(out, tmp);
```

Comment:
Actually, I was thinking maybe we should autogen these ops. For example, if you add `autogen` to `_fw_primal_copy` entry [here](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/native_functions.yaml#L13259), I believe the we can delete this kernel.

Reply:
yeah I wasnt sure if there was some reason they were hand written, maybe they were added before the codegen thing existed. Ill just change the pr to delete and codegen these instead